### PR TITLE
chore: updated api plugins

### PR DESCRIPTION
**Pull request by rosshadden**
Updates API plugins.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,43 +21,43 @@
...
+    "@reactioncommerce/api-plugin-navigation": "~1.1.1",
+    "@reactioncommerce/api-plugin-notifications": "~1.0.1",
+    "@reactioncommerce/api-plugin-orders": "~1.2.1",
+    "@reactioncommerce/api-plugin-payments-example": "~1.1.1",
+    "@reactioncommerce/api-plugin-payments-stripe": "~1.0.1",
+    "@reactioncommerce/api-plugin-payments": "~1.0.1",
+    "@reactioncommerce/api-plugin-pricing-simple": "~1.0.1",
+    "@reactioncommerce/api-plugin-products": "~1.0.1",
+    "@reactioncommerce/api-plugin-settings": "~1.0.1",
+    "@reactioncommerce/api-plugin-shipments-flat-rate": "~1.0.1",
```
Comment (mpaktiti) [inline package.json]: @rosshadden The problem is with this line, if I set it to `"@reactioncommerce/api-plugin-shipments-flat-rate": "~1.0.0"` locally it passes the failing tests. 

We can either wait until we fix this, or rollback this update only and merge the rest.

Comment (rosshadden) [reply package.json]: Good find! I bet it was time-consuming. I'll just change this PR to keep that package at the version you found to work.

Comment (rosshadden) [reply package.json]: @mpaktiti I tried both `~1.0.0` and `1.0.0`, but the tests are still failing. Can you play with making your changes to this branch and try to get it working in CI?

Comment (mpaktiti) [reply package.json]: I set it to `1.0.0` and rebuilt the lock file, tests are passing now ¯\_(ツ)_/¯

I'll register an issue at the plugin's repo (with a note to update this repo's package.json) and look into it.

**Review by mpaktiti — DISMISSED** [review]

**Comment by mpaktiti** [comment]
Issue for the broken dependency https://github.com/reactioncommerce/api-plugin-shipments-flat-rate/issues/5

**Comment by mikemurray** [comment]
@mpaktiti I added fixes, some to this PR, one for shipments based on your ticket, and one for API utils which should address some, hopefully all of broken integration tests.

- https://github.com/reactioncommerce/api-plugin-shipments-flat-rate/pull/6
- https://github.com/reactioncommerce/api-utils/pull/46

**Comment by mpaktiti** [comment]
Looks good @mikemurray, thanks! I updated both packages to latest and all tests pass. 

cc @rosshadden @manueldelreal

**Review by mpaktiti — APPROVED** [review]

**Review by manueldelreal — APPROVED** [review]
My automated tests pass with this one

**Pull request merged**